README.md

```diff
@@ -0,0 +1,35 @@
+# Setup
+
+```bash
+
+docker-compose build
+docker-compose up -d solr-sdr-catalog
+docker-compose run --rm traject bundle install
```

Comment:
I don't know that we need to do the bundle install in the `Dockerfile` quite yet, but we should be aware of the race condition we found on github.com/mlibrary/catalog_solr_index/ with GitHub actions, and use `bundle install --jobs=1` if necessary.

Reply:
The current best pattern is to:

- in development, do `bundle install` as in the README here, so that you don't have to reinstall the entire bundle every time anything changes
- in production, do `bundle install` in `Dockerfile.prod` so that the gems are in the image.